Content.Server/Radio/EntitySystems/RadioDeviceSystem.cs

```diff
@@ -216,9 +216,17 @@ private void OnReceiveRadio(EntityUid uid, RadioSpeakerComponent component, ref
         var name = Loc.GetString("speech-name-relay", ("speaker", Name(uid)),
             ("originalName", nameEv.Name));
 
-        // log to chat so people can identity the speaker/source, but avoid clogging ghost chat if there are many radios
-        var message = args.OriginalChatMsg.Message; // The chat system will handle the rest and re-obfuscate if needed.
-        _chat.TrySendInGameICMessage(uid, message, InGameICChatType.Whisper, ChatTransmitRange.GhostRangeLimit, nameOverride: name, checkRadioPrefix: false, languageOverride: args.Language);
+        {
+            // log to chat so people can identity the speaker/source, but avoid clogging ghost chat if there are many radios
+            var message = args.OriginalChatMsg.Message; // The chat system will handle the rest and re-obfuscate if needed.
+            var chatType = component.IsSpeaker ? InGameICChatType.Speak : InGameICChatType.Whisper;
+            _chat.TrySendInGameICMessage(uid, message, chatType, ChatTransmitRange.GhostRangeLimit, nameOverride: name, checkRadioPrefix: false, languageOverride: args.Language);
+        }
+    }
+
+    private string ToString(RadioChannelPrototype channel)
+    {
+        throw new NotImplementedException();
```

Comment:
This is no longer needed, please remove this.

Reply:
I removed the unneeded code block bracket and moved the indent left one tab

Reply:
The unneeded `ToString()` is still here.

Reply:
I didn't add that

Reply:
It still shows up in the diff -- and will be added if this is merged. If it's not being used, please remove it.

Reply:
gotcha, okay I removed it